crates/pallet-domains/src/lib.rs

```diff
@@ -161,7 +162,7 @@ pub type BlockTreeNodeFor<T> = crate::block_tree::BlockTreeNode<
 >;
 
 /// The current storage version.
-const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
```

Comment:
Is this due to Taurus being on storage version 2 already ?

Reply:
if that is the case, we can manually bump right before we do taurus upgrade no ?